src/rgw/driver/rados/rgw_pubsub_push.cc

```diff
@@ -99,10 +100,17 @@ class RGWPubSubHTTPEndpoint : public RGWPubSubEndpoint {
       ldout(cct, 1) << "ERROR: send failed. http endpoint manager not running" << dendl;
       return -ESRCH;
     }
+    const auto max_inflight = cct->_conf->rgw_http_notif_max_inflight;
+    if (max_inflight != 0 &&
+        s_http_manager_inflight >= cct->_conf->rgw_http_notif_max_inflight) {
```

Comment:
`s_http_manager_inflight >= max_inflight` since you made a local variable for it